tests/testthat/test-bayes.R

```diff
@@ -385,6 +386,10 @@ test_that("retain extra attributes", {
   expect_true(inherits(att$parameters, "parameters"))
   expect_true(inherits(att$metrics, "metric_set"))
 
+  files <- list.files(path = tempdir(), pattern = "^gp_candidates")
```

Comment:
Are you allowed to write to the tempdir when CRAN checks your package? Do you have to clean up after yourself? I'm not sure of the rules here but it might be worth checking

Reply:
I have at least one package that has a function that writes to `tempfile()` and they passed CRAN checks. Don't know if there is a fundamental difference between `tempfile()` and `tempdir()`.

https://github.com/EmilHvitfeldt/wordsalad/blob/master/R/fasttext.R#L67

Reply:
`tune` doesn't run tests on CRAN but win-builder did not flag the use of `tempdir()` so I'll go with it.